server/src/main/java/com/distkv/server/storeserver/Wal.java

```diff
@@ -0,0 +1,45 @@
+package com.distkv.server.storeserver;
+
+import com.distkv.core.LogEntry;
+import com.distkv.core.segment.LogSegment;
+
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * a WAL Implement.
+ * </p>
+ * The interface for wal is not stable now.  it will change for upper logical.
+ * TODO what other methods need to provide?
+ *
+ * @author meijie
+ * @since 0.1.4
+ */
+public class Wal {
+  private LogSegment segment = new LogSegment();
+  private int lastCommitIndex;
+
+  // this used by follower
+  public void append(LogEntry logEntry) {
+    segment.appendLogEntry(logEntry);
+  }
+
+  public void append(byte[] value) {
```

Comment:
Maybe `append(LogEntry logEntry) ` is enough?

Reply:
I will just remove it.